app/helpers/services_helper.rb

```diff
@@ -34,4 +34,21 @@ def path_to_service(service)
   def plugin_language_name
     @service.deployment_option.remove('plugin_')
   end
+
+  def deployment_options
```

Comment:
I do not think this should be a helper most likely something like `Service.active_deployment_options(account)`